packages/datadog-instrumentations/src/helpers/hook.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const path = require('path')
+const iitm = require('../../../dd-trace/src/iitm')
+const ritm = require('../../../dd-trace/src/ritm')
+
+function Hook (modules, onrequire) {
+  if (!(this instanceof Hook)) return new Hook(modules, onrequire)
+
+  this._patched = {}
```

Comment:
can we just preemptively go to Object.create(null) here if we use it as a dictionary

Reply:
I've seen cases where object literals are faster than objects created with `Object.create` when not iterated on. However, in this case it would only affect start time so the trade-off is probably worth it.

Reply:
seems fine either way